substrate/frame/beefy/src/equivocation.rs

```diff
@@ -263,6 +269,14 @@ impl<T: Config> EquivocationEvidenceFor<T> {
 			},
 		}
 	}
+
+	fn hardcoded_slash_fraction(&self) -> Option<Perbill> {
```

Comment:
for `EquivocationOffence` we have now:
field `hardcoded_slash_fraction: Option<Perbill>,`
and function `fn hardcoded_slash_fraction(&self) -> Option<Perbill>`,

at the first look, it seems that this function is meant to return field `hardcoded_slash_fraction` value,
but instead it return hard-coded `Perbill::from_percent(50)`, shouldn't it return `hardcoded_slash_fraction: Option<Perbill>`?

The field and function are disconnected, so anybody can do a silly mistake/typo w/o `()` - where you get totally different value :
```
if let Some(slash_fraction) = self.hardcoded_slash_fraction {
vs
if let Some(slash_fraction) = self.hardcoded_slash_fraction() {
```

Can you please add some doc why hard-coded `Some(Perbill::from_percent(50)` here?

Reply:
ok, sorry, ignore my previous comment, I see now that the function is a different struct `EquivocationEvidenceFor`